config/default/field.field.paragraph_row.hs_flexible_page.field_hs_flexible_components.yml

```diff
@@ -58,13 +61,25 @@ settings:
       hs_spotlight:
         weight: 21
         enabled: false
+      hs_testimonial:
+        weight: 28
+        enabled: false
       hs_text_area:
         weight: 22
         enabled: false
+      hs_timeline:
+        weight: 30
+        enabled: false
+      hs_timeline_item:
+        weight: 31
+        enabled: false
```

Comment:
Note to self. Check to see if this file was included in the timeline work. Because this makes it look like it wasn't.

Reply:
Remove everything that isn't hs_gradient_hero. See latest config on the release branch: https://github.com/SU-HSDO/suhumsci/blob/926300c95dbf7ac1dc5ac13c9efb34f8ff9001dd/config/default/field.field.paragraph_row.hs_flexible_page.field_hs_flexible_components.yml